### PR TITLE
Final monitoring doc revision

### PR DESCRIPTION
**Pull request by youngbupark**
# Description

Revise monitoring docs

## Issue reference

#431 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: concepts/observability/metrics.md [hunk 1]
```diff
@@ -1,30 +1,21 @@
...
 ## Metrics
-Dapr will expose metrics for Darp's HTTP pipeline, gPRC pipeline, Go runtime, metrics server and process.
-> Note: The gRPC metrics are all initialized with zero values and are therefore immediately visible on the 
-metrics endpoint. HTTP metrics are not zero initialized so will only become visible once a HTTP request has
-passed through the HTTP pipeline.
 
-## Prometheus
-To consume these metrics you'll need to install Prometheus or something compatible with scraping Prometheus
-metrics.
+Each Dapr system processe emits Go runtime and process emtrics by default and have their own metrics:
```
Comment (pruthvidhodda) [inline concepts/observability/metrics.md]: processe -> process
emtrics -> metrics

File: howto/setup-monitoring-tools/setup-prometheus-grafana.md [hunk 2]
```diff
@@ -117,21 +117,29 @@ kibana-kibana                        ClusterIP   10.0.188.224   <none>        56
...
+8. Import Dapr dashboards.
 
 You can now import built-in [Grafana dashboard templates](../../reference/dashboard/README.md).
 
 Refer [here](../../reference/dashboard/README.md) for details.
 
 ![upload json](./img/grafana-uploadjson.png)
 
-You can find screenshots of Dapr dashboards [here](../../reference/dashboard/img/).
+9. Find the dashboard that you imports and enjoy!
```
Comment (shalabhms) [inline howto/setup-monitoring-tools/setup-prometheus-grafana.md]: nit : imported

File: concepts/observability/metrics.md (hunk 1 above)
Comment (youngbupark) [reply concepts/observability/metrics.md]: Fixed thanks!

File: howto/setup-monitoring-tools/setup-prometheus-grafana.md (hunk 2 above)
Comment (youngbupark) [reply howto/setup-monitoring-tools/setup-prometheus-grafana.md]: Fixed!

**Pull request merged**